app/frontend/wallet/shelley-wallet.ts

```diff
@@ -282,7 +301,7 @@ const ShelleyWallet = ({config, randomInputSeed, randomChangeSeed, cryptoProvide
     txType?: string
   }
 
-  const utxoTxPlanner = async (args: utxoArgs, accountAddress: string) => {
+  const utxoTxPlanner = async (args: utxoArgs, accountAddressPath) => {
     const {address, coins, donationAmount, poolHash, stakingKeyRegistered, txType} = args
     const changeAddress = await getChangeAddress()
     const availableUtxos = await getUtxos()
```

Comment:
Line 308 is confusing. Pointer addresses are staking, so why are they included under nonStakingUtxos? Or do we assume no pointer addresses whatsoever? What happens when a user has an existing account which already contains pointer addresses? I guress this does not affect the user at all, but still, either the naming is wrong or based on some silent assumption I do not know.